install.md

```diff
@@ -282,9 +282,13 @@ To configure the installation settings:
 
    Where:
 
-   - `ADMIN-USERNAME` is the name of an existing user to grant system admin privileges. You can specify as many users as you want, one per line. These names should be identifiers for user accounts, not Kubernetes service accounts.
+   - `ADMIN-USERNAME` is the name of an existing user to grant system admin privileges. You can specify as many users as you want, one per line. These names should be identifiers for Kubernetes user accounts, not Kubernetes service accounts.
+     - See additional optional values in the following example.
```

Comment:
I'm not sure this belongs as a sub-bullet under the `ADMIN-USERNAME` config since it's related to the section below about viewing all optional values.

Reply:
Moved it down while resolving merge conflicts in this area.